spdx-utils/build.gradle.kts

```diff
@@ -358,7 +358,10 @@ val generateLicenseRefTextResources by tasks.registering {
         licensesDir.walk().maxDepth(1).filter {
             it.isFile && it.extension == "yml" && !it.nameWithoutExtension.endsWith("-exception")
         }.forEach { file ->
-            val isSpdxLicense = file.readLines().any { it.startsWith("spdx_license_key: ") }
+            val isSpdxLicense = file.readLines().any {
+                it.startsWith("spdx_license_key: ") && !it.substringAfter("spdx_license_key: ").trim()
+                    .startsWith("LicenseRef-")
```

Comment:
Do you think it's safe enough to simplify this to just `it.startsWith("spdx_license_key: ") && !it.contains(" LicenseRef-")`

Reply:
Famous last words....will do it ;-)